<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bool_widget</class>
 <widget class="QWidget" name="bool_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>114</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>96</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
